File: 1328-break-a-palindrome/1328-break-a-palindrome.cpp
class Solution {
public:
    string breakPalindrome(string p) {
        int n=p.size();
        if(n==1){
            return "";
        }
        for(int i=0;i<n/2;i++){
            if(p[i]!=97){
                p[i]=97;
                return p;
            }
        }
        p[n-1]=98;
        return p;
    }
};

File: 1328-break-a-palindrome/1328-break-a-palindrome.java
class Solution {
    public String breakPalindrome(String p) {
        int n=p.length();
        if(n==1)
            return "";
        StringBuilder s = new StringBuilder(p);
        for(int i=0;i<n/2;i++){
            if(s.charAt(i)!='a'){
                s.setCharAt(i,'a');
                return s.toString();
            }
        }
        s.setCharAt(n-1,'b');
        return s.toString();
    }
}